### ISSUE TITLE
clvk namespace

### ISSUE THREAD

**Issue by jrprice** [kpet/clvk#128]
I'm hitting some build issues on a platform whereby some of the symbols that clvk defines in the global namespace are clashing with those defined elsewhere (e.g. `string`).

I'd recommend putting everything in top-level `clvk` namespace to avoid these sorts of issues. @kpet Would you accept a patch to do this?

**Comment by kpet** [comment]
Yeah, this one was not the smartest name choice. I've pushed a change to rename it.

Yes, I'd been thinking about this, go for it. I'd rather we don't rename types and remove prefixes though, at least for now.
